rktl_msgs/msg/MatchStatus.msg

```diff
@@ -1,4 +1,4 @@
-Header header
+# Header header
```

Comment:
Something might use this, so let's not remove it.

Reply:
OK